rx.wamp/rxwamp.d.ts

```diff
@@ -0,0 +1,47 @@
+// Type definitions for rx.wamp 0.5.0
+// Project: https://github.com/paulpdaniels/rx.wamp/
```

Comment:
see http://definitelytyped.org/guides/contributing.html#naming-the-file for more details.